js/components/utils.js

```diff
@@ -376,6 +376,28 @@ var Utils = {
     return this.isAndroid() || this.isIos();
   },
 
+
+  /**
+   * Get type of user device.
+   *
+   * @returns {string} - name of the user device, may be one of the values 'desktop', 'phone' or 'tablet'.
+   */
+  getUserDevice: function() {
```

Comment:
Would recommend using the existing `Utils.isIPhone()` function if possible.

Reply:
oh, in this function we don't determine if a device is iPhone, but determine if the device is mobile phone or tablet, or desktop.

Reply:
@jpfranco ok, i got your point. I will change the code. Thanks!

Reply:
@jpfranco I checked sizes for devices and I undestood that a smaller device may have bigger size in px than bigger one. http://screensiz.es/ . That means that a tablet may be checked as a phone and a phone as a tablet.

So, now I think that if we want to check if device is phone, it would be better to use userAgent (as we do in the function).
Or maybe you have any other ideas?

Reply:
Oh ok I see. So in some cases the resolution is higher than the screen size. I think we can check the user agent then. Thanks for updating all the old unit tests by the way.